README.md

```diff
@@ -5,11 +5,16 @@ high-performance mechanism for publishing the revocation status of Verifiable
 Credentials.
 
 We encourage contributions meeting the [Contribution
-Guidelines](CONTRIBUTING.md).  While we prefer the creation of issues
+Guidelines](CONTRIBUTING.md). While we prefer the creation of issues
 and Pull Requests in the GitHub repository, discussions often occur
 on the
 [public-credentials](http://lists.w3.org/Archives/Public/public-credentials/)
 mailing list as well.
 
 ### Other useful links
-* [Public group email archive](https://lists.w3.org/Archives/Public/public-credentials/)
+
+- [Public group email archive](https://lists.w3.org/Archives/Public/public-credentials/)
+
+### Specification Status
+
+- This specification has been superceeded by [VC Status List 2021](https://w3c-ccg.github.io/vc-status-list-2021) and will no longer be maintained
```

Comment:
```suggestion
- This specification has been superseded by [VC Status List 2021](https://w3c-ccg.github.io/vc-status-list-2021) and will no longer be maintained
```

Reply:
Sentence should also get a concluding "."....